scripts/exglobal_atmos_analysis.sh

```diff
@@ -812,13 +814,17 @@ OBS_INPUT::
    prepbufr       ps          null        ps                  0.0     0     0
    prepbufr       t           null        t                   0.0     0     0
    prepbufr_profl t           null        t                   0.0     0     0
+   hdobbufr       t           null        t                   0.0     0     0
    prepbufr       q           null        q                   0.0     0     0
    prepbufr_profl q           null        q                   0.0     0     0
+   hdobbufr       q           null        q                   0.0     0     0
    prepbufr       pw          null        pw                  0.0     0     0
    prepbufr       uv          null        uv                  0.0     0     0
    prepbufr_profl uv          null        uv                  0.0     0     0
    satwndbufr     uv          null        uv                  0.0     0     0
+   hdobbufr       uv          null        uv                  0.0     0     0
    prepbufr       spd         null        spd                 0.0     0     0
+   hdobbufr       spd         null        spd                 0.0     0     0
```

Comment:
Addition of hdobbufr spd prompted me to look at exglobal_diag.sh.   diagtype[0] lists the obs types to include in the cnvstat file.  conv_spd is not listed.  If we process hdobbufr spd but do not list conv_spd in diagtype[0], the hdobbufr spd diagnostics will not be included in cnvstat.   Please add conv_spd to exglobal_diag.sh.

Reply:
Added. Committed changes.